views/components/Graphs/BarGraph.jsx

```diff
@@ -28,26 +38,32 @@ const BarGraph = ({ graph }) => {
         labelComponent={<VictoryLabel />}
       />
       <VictoryAxis label='Site' style={graphStyles.xAxis} />
-      <VictoryAxis label='Total' dependentAxis style={graphStyles.yAxis} />
+      <VictoryAxis
+        label='Total'
+        dependentAxis
+        style={graphStyles.yAxis}
+        tickFormat={(yAxisValue) => `${yAxisValue}%`}
+      />
       <VictoryStack>
         {Object.values(graph.data).map((data, idx) => (
           <VictoryBar
             data={data}
             x='study'
-            y='count'
+            y='percent'
             key={'bar' + idx}
             style={{
               data: {
                 fill: ({ datum }) => datum.color,
               },
             }}
-            labels={({ datum: { count, studyTarget, study } }) =>
-              `Study: ${study} \n Current: ${count} \n Target: ${studyTarget}`
+            labels={({ datum }) =>
+              !!datum.percent ? `${datum?.percent}%` : null
```

Comment:
If datum is potentially undefined should this be 
```suggestion
              !!datum?.percent ? `${datum.percent}%` : null
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/125/commits/825202fb4464e2ca01467a863b7d1e14823e6808

Reply:
👍 we don't need to check datum twice
```diff
-!!datum?.percent ? `${datum?.percent}%` : null
+!!datum?.percent ? `${datum.percent}%` : null
```

Reply:
resolved in 0f48dd93ba9f23840b3e0331329de77eb7fabd5e